README.md

```diff
@@ -1,101 +1,62 @@
-# <repo name>
+# quic-reverse-http-tunnel
 
 [![reuse compliant](https://reuse.software/badge/reuse-compliant.svg)](https://reuse.software/)
```

Comment:
```suggestion
[![REUSE status](https://api.reuse.software/badge/github.com/gardener/quic-reverse-http-tunnel)](https://api.reuse.software/info/github.com/gardener/quic-reverse-http-tunnel)
```
Otherwise the badge would be fake news 😄